src/phase0/mod.rs

```diff
@@ -29,9 +29,10 @@ pub use state_transition::*;
 pub use validator::*;
 
 pub const BASE_REWARDS_PER_EPOCH: u64 = 4;
-pub const DEPOSIT_CONTRACT_TREE_DEPTH: usize = 2usize.pow(5);
+pub const DEPOSIT_CONTRACT_TREE_DEPTH: usize = 2usize.saturating_pow(5);
```

Comment:
seems unnecessary? if there is a compatibility issue with `pow` then we can just write `32usize` here